docs/_includes/head.html

```diff
@@ -9,6 +9,7 @@
     <script>
         var site = {}; // site settings
     </script>
+    <script src="https://embed.runkit.com"></script>
```

Comment:
```suggestion
```
I understand you probably moved this here so that the function in `embed-scripts.html` above wouldn't crash due to a undefined `RunKit` variable, but this will block rendering, so we should try to find a way of keeping this in the footer.